cmd/jaeger/internal/integration/grpc_test.go

```diff
@@ -33,7 +33,8 @@ func TestGRPCStorage(t *testing.T) {
 
 	s := &GRPCStorageIntegration{
 		E2EStorageIntegration: E2EStorageIntegration{
-			ConfigFile: "../../config-remote-storage.yaml",
+			ConfigFile:         "../../config-remote-storage.yaml",
+			SkipStorageCleaner: true,
```

Comment:
@yurishkuro Is this okay to do? grpc factory doesn't implement Purger so it was causing the tests to fail

Reply:
but how did it work before?

Reply:
If a storage didn't implement the purger factory - the extension would still be able to start a 500 would be returned upon trying to purge (https://github.com/jaegertracing/jaeger/blob/main/cmd/jaeger/internal/integration/storagecleaner/extension.go#L56). There's a slight change in behaviour now so that the extension doesn't start if the Purger isn't implemented.